west.yml

```diff
@@ -66,7 +66,7 @@ manifest:
       revision: b280eec5d3b1296b231117c1999bcd0269b6ecc4
       path: modules/hal/microchip
     - name: hal_silabs
-      revision: be39d4eebeddac6e18e9c0c3ba1b31ad1e82eaed
+      revision: pull/13/head
```

Comment:
Personally, I don't understand this change.

Reply:
As per the guidelines in https://docs.zephyrproject.org/latest/guides/modules.html#changes-to-existing-module, 13 in the example above indicated the pull request number submitted to the hal_silabs repository. Once the module changes are reviewed and merged, the revision is changed to the commit hash from the module repository.

Reply:
Really thanks, something I don't know before:)